### PR TITLE
M13 documentation updates

### PR DESCRIPTION
**Pull request by mit-mit**
Documentation updates for https://github.com/flutter/flutter-intellij/issues/854

### CODE REVIEW

**Comment by mit-mit** [comment]
@skybrian please review the android edit commit

File: faq.md [hunk 1]
```diff
@@ -301,9 +301,9 @@ a runtime cost.  The "slow mode" banner indicates that these checks are enabled.
 You can run your app without these checks by using either `--profile` or `--release`
 flag to `flutter run`.
 
-If you are using the Flutter plugin for IntelliJ, please note that it currently
-does not support running in `--profile` or `--release` modes (see
-[issue #410](https://github.com/flutter/flutter-intellij/issues/410)).
+If you are using the Flutter plugin for IntelliJ, you can launch the app in
+profile or release mode using the menu entries **Run > Flutter Run (Profile)**
```
Comment (devoncarew) [inline faq.md]: 🎆

File: intellij-ide.md [hunk 2]
```diff
@@ -188,8 +188,22 @@ types of code changes cannot be hot reloaded though:
 For these changes you can fully restart your application, without having to end
 your debugging session:
 
-1. In the Debug panel at the bottom, click the **Full Application Restart button**
-(shown with an icon containing a circular arrow).
+1. Don't click the Stop button; simply re-click the Run button (if in a run
+session) or Debug button (if in a debug session).
```
Comment (devoncarew) [inline intellij-ide.md]: Here (or elsewhere), we'll want to document how to access the full restart functionality. Currently accessed via:

- shift-click on the hot reload button instead performs a full restart
- hitting the 'run' or 'debug' button while an app is currently running will restart it

And perhaps, a brief blurb on the differences between hot reload (patches changes but maintain state) and full restart (restart the isolate running the app, slower but resets the app's state).

**Comment by devoncarew** [comment]
lgtm, w/ some comments about documenting how to access the restart functionality

File: intellij-ide.md [hunk 3]
```diff
@@ -188,8 +188,22 @@ types of code changes cannot be hot reloaded though:
...
+To enable editing of Android code in IntelliJ IDEA, you need to configure the
+location of the Android SDK:
+
+1. Right-click the **android** folder in the Project view, and select **Open
+Module Settings**.
+1. In **Sources** tab, locate the **Language level** field, and select level '8'
+or later.
+1. In **Dependencies** tab, locate the **Module SDK** field, and select an
+Android SDK. If no SDK is listed, click **New...** and specify the location of
+the Android SDK.
```
Comment (skybrian) [inline intellij-ide.md]: "This should be the same Android SDK shown by 'flutter doctor' in the 'Android toolchain' section."

(Or something like that. Do we need to explain how to run flutter doctor?)

**Comment by skybrian** [comment]
lgtm

**Comment by mit-mit** [comment]
Reverting FAQ changed due to https://github.com/flutter/flutter-intellij/issues/930. New PR with that change is https://github.com/flutter/website/pull/511.

File: intellij-ide.md (hunk 2 above)
Comment (mit-mit) [inline intellij-ide.md]: Does the current explanation of the differences (~ 5 lines above your comment) not explain reload vs restart?

Comment (devoncarew) [inline intellij-ide.md]: ah, I see that now; looks good

**Comment by devoncarew** [comment]
M13 is published - @mit-mit I'll land this so the website is up-to-date

**Pull request merged**